cozepy/conversations/message/feedback/__init__.py

```
@@ -0,0 +1,111 @@
+
+
+class FeedbackType(str, Enum):
+    """
+    反馈类型
+    """
+
+    LIKE = "like"  # 赞
+    UNLIKE = "unlike"  # 踩
+
+
+class CreateConversationMessageFeedbackResp(CozeModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)